lib-rs/src/lib.rs

```diff
@@ -27,7 +109,34 @@ pub unsafe fn slice_from_raw_parts<'a, T>(ptr: *const T, len: usize) -> &'a [T]
     }
 }
 
-/// Wraps a non-owning FFI string
+/// Wraps a non-owning FFI string without the overhead of CString/String
```

Comment:
I would say non-owned, but does `owning` have some special meaning?

Reply:
Changed to `owned`